spread.yaml

```diff
@@ -816,6 +824,42 @@ suites:
         restore: |
             "$TESTSLIB"/prepare-restore.sh --restore-suite
 
+    tests/nested/classic-preseed/:
```

Comment:
Note, define new nested suite to control VM start, which normally happens in prepare for other nested tests. CC @sergiocazzolato